### ISSUE TITLE
Export/Import solution under Operations menu

### ISSUE THREAD

**Issue by dvXcode** [Azure/api-management-developer-portal#650]
## Expected behavior

It will be nice to have ability to import and export portal configuration from Developer Portal. 

## Is your portal managed or self-hosted?

Managed

![ImportExportMenuDPpng](https://user-images.githubusercontent.com/51912361/83136460-cfa7ed80-a0e7-11ea-82c0-de07b451334a.png)

**Comment by azaslonov** [comment]
Thank you for suggestion, we'll consider this option.

**Comment by jnancel** [comment]
Yes, extremely useful feature for us too, we have several customers that are using several developer portals on different environments, and it's quite painful to have to set the same UI of your portal several times. If it was also doable through an API call, it would be awesome so that we can integrate that in our CI/CD pipelines.

Users would then customize their portal once on dev, and then we'll be able to roll out that customization on the different environments easily.

Thanks
